ovs/flow.go

```diff
@@ -262,6 +266,11 @@ func (f *LearnedFlow) MarshalText() ([]byte, error) {
 	b = append(b, ","+finHardTimeout+"="...)
 	b = strconv.AppendInt(b, int64(f.FinHardTimeout), 10)
 
+	b = append(b, ","+hardTimeout+"="...)
```

Comment:
should these be added only conditionally that have valid values?

Reply:
so I followed the pattern the other options are using which is if the value is not specified, it will be set to 0 by default, hence we will have hard_timeout=0 and limit=0 explicitly stated in the created flow. Or did you have something else in mind ?

Reply:
when I say the "created flow" I mean the "the generated text by go-openvswitch that represent the flow". Once the flow has been inserted in ovs with hard_timeout=0 and limit=0, these options will not show up in the "dump", since '0' is equal of not using them at all

Reply:
I understand the default behavior when these are not specified is as before. I was suggesting we do sanitize the input as general good practice and skip the operations below if the values are indeed 0, which preserves the old behavior prior to this commit. Consider it more of a nit and I expect no action if you don't consider it useful.

Reply:
gotcha ! yeah I think I will leave it as is for now, thanks :)